felix/bpf/mock/multiversion/v5/map.go

```diff
@@ -23,15 +23,14 @@ import (
 )
 
 var MockMapParams = bpf.MapParameters{
-	Filename:     "/sys/fs/bpf/tc/globals/cali_mock",
-	Type:         "hash",
-	KeySize:      40,
-	ValueSize:    88,
-	MaxEntries:   1024,
-	Name:         "cali_mock",
-	Flags:        unix.BPF_F_NO_PREALLOC,
-	Version:      5,
-	UpdatedByBPF: true,
```

Comment:
I thought we needed this to emulate the maps being updated by bpf :thinking:  Is it related to the flake? Imo not.

Reply:
If we have ```UpdatedByBPF``` set to ```true```, we don't close the oldfd and do that in the other flow. We don't call that flow in the UTs in some of the tests. As a result, there is a reference to the map present even after the test which results in an error. Other option is to close the ```oldfd``` in ```Close()```, so that all the references are closed properly.

Reply:
That might be the way to go. We want to test the "real thing" rather than to test what is easier to test ;-) Close() can do a test whether the oldfd is still valid.